crates/schemalate/src/firebolt/firebolt_schema_builder.rs

```diff
@@ -57,7 +57,7 @@ fn build_shape_from_schema(schema_str: &str) -> Result<Shape, Error> {
 pub fn build_firebolt_schema(binding: &Binding) -> Result<TableSchema, Error> {
     let fs = binding.field_selection.as_ref().unwrap();
     let projections = &binding.collection.as_ref().unwrap().projections;
-    let schema_str = &binding.collection.as_ref().unwrap().schema_json;
+    let schema_str = &binding.collection.as_ref().unwrap().write_schema_json;
```

Comment:
Is the write schema the correct thing to be using here? If I understand correctly it seems like the possibly-more-restrictive read schema of the collection is what we'd want to reflect in a materialization.

Reply:
Done